api/handlers/containerhandler.py

```diff
@@ -67,7 +67,10 @@ class ContainerHandler(base.RequestHandler):
             'storage_schema_file': 'session.json',
             'payload_schema_file': 'session.json',
             # Remove subject first/last from list view to better log access to this information
-            'list_projection': {'info': 0, 'analyses': 0, 'subject.firstname': 0, 'subject.lastname': 0},
+            'list_projection': {'info': 0, 'analyses': 0, 'subject.firstname': 0,
+                                'subject.lastname': 0, 'subject.sex': 0, 'subject.age': 0,
+                                'subject.race': 0, 'subject.ethnicity': 0, 'subject.info': 0,
+                                'file.info': 0, 'session.tag': 0, 'acquisition.tag': 0},
```

Comment:
This is the list projection specific to a session object mongo object, so `tags` would be the key. 
`file.info` should be `files.info`